src/utils/tokenGenerator.utils.ts

```diff
@@ -18,3 +18,19 @@ const generateToken = async (user: UserAttributes) => {
   );
 };
 export { generateToken };
+
+
+
+export const generateResetToken = (user: UserAttributes) => {
```

Comment:
Use generateToken instead of creating new function works the same with existing one

Reply:
> Use generateToken instead of creating new function works the same with existing one

It has been corrected